### PR TITLE
update user schema

### PR DESCRIPTION
**Pull request by yuhwaa**

### CODE REVIEW

File: models/User.js [hunk 1]
```diff
@@ -2,9 +2,14 @@ const bcrypt = require("bcrypt");
 const mongoose = require("mongoose");
 
 const UserSchema = new mongoose.Schema({
-  userName: { type: String, unique: true },
+  userId: { type: String, unique: true },
```
Comment (JoeDravarol) [inline models/User.js:5]: @intelagense Do we need to explicitly add a separate userID field? I think MongoDB/Mongoose will by default generate an *_id* field for every schema.

Comment (Caleb-Cohen) [reply models/User.js:5]: This should be correct. @yuhwaa I can edit this out unless you disagree.

**Review by Caleb-Cohen — CHANGES_REQUESTED** [review]
userId should be removed.

File: models/User.js (hunk 1 above)
Comment (intelagense) [reply models/User.js:5]: That is right @JoeDravarol that we do not need to add one. I meant to use that auto-generated field for the ID of the user. It is not labeled clearly on the whiteboard.

**Pull request merged**